packages/flutter/lib/src/material/data_table.dart

```diff
@@ -311,8 +313,20 @@ class DataTable extends StatelessWidget {
   /// row is selectable.
   final ValueSetter<bool> onSelectAll;
 
+  /// The height of each row (excluding the row that contains column headings).
+  ///
+  /// This value is optional and defaults to 48.0 if not specified.
+  final double dataRowHeight;
+
+  /// The height of the heading row.
+  ///
+  /// This value is optional and defaults to 56.0 if not specified.
```

Comment:
I think this (and the similar line for `dataRowHeight`) are slightly misleading.  Maybe it should say "This value is optional, and if not specified, 56.0 will be used".  The reason is that as you wrote the code, `headingRowHeight` actually is not changed to the default value, so if anyone tries to look at it from outside of this class, they would get null and not the default value.

Or, if you think it makes sense, you could keep this comment and instead change it so that the value of `headingRowHeight` is actually changed to 56.0 if null is given.

Reply:
Took the approach mentioned in https://github.com/flutter/flutter/pull/33628#discussion_r292634315